arquillian/arquillian-tomee-common/pom.xml

```diff
@@ -86,6 +86,16 @@
       <artifactId>xbean-finder-shaded</artifactId>
       <version>${xbeanVersion}</version>
       <scope>provided</scope>
+      <exclusions>
```

Comment:
I am wondering:

Is this needed?  Afaik [xbean-finder-shaded](https://mvnrepository.com/artifact/org.apache.xbean/xbean-finder-shaded/4.18) only has a compile dependency towards `xbean-asm9-shaded`. The `org.ow2.asm` stuff is a transient dependency of `xbean-asm9-shaded`. So shouldn't we exclude  `xbean-asm9-shaded` here as it comes from `openwebbeans` anyway?

Reply:
ow2 deps should be excluded in root pom in dependency management of xbean-asm9-shaded dependency declaration to ensure the whole tree gets it for free.

Reply:
Ok, let me change it tomorrow. Thanks for the hint, didn't really now where to exclude or if all exclusions were needed

Reply:
Changed it, only that exclusion is enough